contracts/src/v0.8/ccip/FeeQuoter.sol

```diff
@@ -57,10 +57,11 @@ contract FeeQuoter is AuthorizedCallers, IFeeQuoter, ITypeAndVersion, IReceiver,
   event DestChainConfigUpdated(uint64 indexed destChainSelector, DestChainConfig destChainConfig);
   event DestChainAdded(uint64 indexed destChainSelector, DestChainConfig destChainConfig);
 
-  /// @dev Token price data feed configuration
+  /// @dev Token price data feed configuration. Represents both Keystone report and Data Feed configurations.
   struct TokenPriceFeedConfig {
-    address dataFeedAddress; // ──╮ AggregatorV3Interface contract (0 - feed is unset)
-    uint8 tokenDecimals; // ──────╯ Decimals of the token that the feed represents
+    address dataFeedAddress; // ──╮ AggregatorV3Interface contract (0 - Data Feed is unset)
```

Comment:
Let's add comments explaining what a zero data feed address implies. The comment above is better but there's still some room for improvement